.gitignore

```diff
@@ -183,9 +183,6 @@
 
 # ------------------------------------------------------------------------------
 
-# We want all custom site settings in settings.local.php
-!/web/sites/*/settings*.php
-
```

Comment:
I don't understand why this change is needed in relation to the other changes in this pull request. Can you elaborate?